pkg/maps/authmap/cell.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package authmap
+
+import (
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/option"
+)
+
+// Cell provides the auth.Map which is responsible for interacting with the eBPF map auth.
+var Cell = cell.Module(
+	"auth-map",
+	"eBPF map which manages authenticated connections between identities",
+
+	cell.Provide(newAuthMap),
+	cell.Invoke(func(Map) {}),
+)
+
+func newAuthMap(lifecycle hive.Lifecycle) (Map, error) {
+	authMap := newMap(option.Config.AuthMapEntries)
```

Comment:
i decided against a dedicated config for the cell and went with the "global one". the reason is that the value is still used in other places (nodeconfig, bpf map status, ...). IMO this should be addressed separately.